src/lib/messenger/background.ts

```diff
@@ -282,21 +259,21 @@ const setupBackground = async (
 
 				observer.observe(backgroundParent);
 
-				const creativeInfo = getCreativeId();
+				const advert = getAdvertById(adSlot.id);
 
-				if (creativeInfo) {
-					initVideoProgressReporting(
-						creativeInfo.creativeId,
-						creativeInfo.lineItemId,
-					);
-				}
+				const shouldReportVideoProgress =
+					advert?.creativeTemplateId === 11885667;
```

Comment:
Maybe make this ID a constant to help explain what it is?

Reply:
Good shout